src/atomvm_uf2create_provider.erl

```diff
@@ -32,13 +29,15 @@
 -define(PROVIDER, uf2create).
 -define(DEPS, [packbeam]).
 -define(OPTS, [
+    {family_id, $f, "family_id", string, "Flavor of uf2 file to create (default rp2040)"},
     {output, $o, "output", string, "Output path/name"},
     {start, $s, "start", string, "Start address for the uf2 binary (default 0x10180000)"},
     {input, $i, "input", string, "Input avm file to convert to uf2"}
 ]).
 
 -define(DEFAULT_OPTS, #{
-    start => "0x10180000"
+    start => os:getenv("ATOMVM_PICO_APP_START", "0x10180000"),
+    family_id => os:getenv("ATOMVM_PICO_UF2_FAVOR", rp2040)
```

Comment:
I guess you mean FLAVOR and not FAVOR ;-)

Reply:
Yes! Thank you!

Reply:
Actually, I should make that _FAMILY. I changed the name to match the uf2tool, and picotool parameter name.